mkdocs.yml

```diff
@@ -0,0 +1,31 @@
+site_name: RESTier Documentation
+site_description: How to use the RESTier framework for .NET.
+theme: readthedocs
+pages:
+- Home:
+    - 'Introduction': 'index.md'
+    - 'Getting Started': 'getting-started.md'
```

Comment:
I took the numbers out because the ASP.NET team doesn't use them, and the documentation needs to be fairly consistent across the total API surface. Plus, numbers are just one more thing you have to keep up with changing as the docs evolve and new sections are added. I just don't think the benefit is worth the effort.